lib/fog/aws/models/elb/policy.rb

```diff
@@ -45,12 +44,8 @@ def destroy
           reload
         end
 
-        def reload
-          load_balancer.reload
-        end
-
         def load_balancer
-          collection.load_balancer
+          service.load_balancers.new(identity: collection.load_balancer_id)
```

Comment:
Yeah, stuff like this is starting to bite us more and more often. Breaking changes just also are, well, breaking, so caution prevails. Would love to cut it out soon though.